extensions/wrapper/wrapper-ee-api/src/main/java/de/sovity/edc/ext/wrapper/api/ee/EnterpriseEditionResource.java

```diff
@@ -54,82 +51,80 @@ public interface EnterpriseEditionResource {
-                         content = @Content(schema = @Schema(implementation = StoredFile.class))),
-            @ApiResponse(responseCode = "400", description = "Invalid request"),
-            @ApiResponse(responseCode = "500", description = "Internal server error")
-    })
     StoredFile uploadStoredFile(
-            @Parameter(description = "The name of the file.",
-                       required = true,
-                       style = ParameterStyle.FORM
+            @Parameter(
+                    description = "The name of the file.",
+                    required = true,
+                    style = ParameterStyle.FORM
             ) String fileName,
 
-            @Parameter(description = "The file content to upload.",
-                       required = true,
-                       style = ParameterStyle.FORM
+            @Parameter(
+                    description = "The file content to upload.",
+                    required = true,
+                    style = ParameterStyle.FORM
             ) byte[] fileContent
     );
 
-    //ToDo: this API endpoint might be extended with query parameters in the future.
     @GET
     @Path("file-storage/stored-files")
     @Produces(MediaType.APPLICATION_JSON)
-    @Operation(summary = "Get all uploaded files.",
-               description = "Get all uploaded files from the file storage.")
-    @ApiResponses(value = { @ApiResponse(responseCode = "200",
-                                         description = "Successfully processed the list of StoreFile objects.",
-                                         content = @Content(array = @ArraySchema(schema = @Schema(implementation = StoredFile.class)))),
-                            @ApiResponse(responseCode = "400", description = "Invalid request"),
-                            @ApiResponse(responseCode = "500", description = "Internal server error")
-    })
+    @Operation(
+            summary = "Get all stored files.",
+            description = "Get all files stored in file storage"
+    )
     List<StoredFile> listStoredFiles();
 
     @POST
     @Path("file-storage/{storedFileId}/assets")
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
-    @Operation(summary = "Create the file storage asset.",
-               description = "Create the file storage asset for the given stored file id.")
-    @ApiResponses(value ={@ApiResponse(responseCode = "200",
-                                       description = "File store asset successfully created.",
-                                       content = @Content(schema = @Schema(implementation = StoredFile.class))),
-                          @ApiResponse(responseCode = "400", description = "Invalid request"),
-                          @ApiResponse(responseCode = "500", description = "Internal server error")
-    })
+    @Operation(
+            summary = "Create the file storage asset.",
```

Comment:
Minimal wording, we are not creating "the file storage asset", but some file storage asset.

"Create a file storage asset" is better

"Create an asset from a stored file." also a candidate